<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>替换</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>查找内容(N):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>15</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblReplace">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>替换为(P):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="replaceEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>15</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>查找下一个(F)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>替换(R)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceAllButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>105</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>全部替换(A)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cbCaseSensitive">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>区分大小写</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
